### PR TITLE
Upgrade Qute Web to 3.0.2

### PR DESCRIPTION
**Pull request by mkouba**

### CODE REVIEW

File: generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-fop/pom.xml [hunk 1]
```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>org.apache.pdfbox</groupId>
       <artifactId>pdfbox</artifactId>
-      <version>3.0.2</version>
```
Comment (mkouba) [inline generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-fop/pom.xml:60]: This is not correct but should be fine: https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/quarkus-platform.3A.20weird.20upgrade.20issue

**Comment by mkouba** [comment]
CC @aloubyansky @gsmet

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
I included it in the 3.12.2 PR.

**Comment by mkouba** [comment]
> I included it in the 3.12.2 PR.

Thanks!

**Pull request merged**